### ISSUE TITLE
TypeError: Cannot read property 'toString' of undefined after upgrading to the latest release

### ISSUE THREAD

**Issue by mairh** [FormidableLabs/radium#537]
Shouldn't `Radium` just throw a warning instead of error when passing null or undefined as the value for some property. I am getting the following error

<img width="554" alt="screen shot 2016-01-18 at 16 13 28" src="https://cloud.githubusercontent.com/assets/4072250/12393498/bbc5ec0c-bdfe-11e5-873a-de3ea6492bdf.png">

**Comment by ianobermiller** [comment]
Ugh. Yes, good catch. I'll add a unit test so we don't break this again.

**Comment by ianobermiller** [comment]
Actually, looks like this was broken by `inline-style-prefixer`. cc @rofrischmann

**Comment by rofrischmann** [comment]
Hmm, How comes there is null or undefined as a value? I need to transform the value to a string to check for some content. I will add another check then.

**Comment by rofrischmann** [comment]
I am now checking directly for `typeof value === 'string'`. This prevents the calc pugin from breaking if `false` or `undefined` is passed in. ( See Version 0.6.6 )

**Comment by ianobermiller** [comment]
> Hmm, How comes there is null or undefined as a value?

Very common if you are conditionally assigning a prop:

```jsx
const style = {
  height: '100px',
  width: typeof this.props.width === 'number' ? this.props.width : undefined,
};
```

You might also get `false`, say if you did `typeof this.props.width === 'number' && this.props.width`.

**Comment by rofrischmann** [comment]
Ah I see. Always considered thought one should not do this. I always try to set defaultProps :P But seems legit though. Thanks for explaining :)

**Comment by mairh** [comment]
Guys, any idea when you should able to do a new release with the bug fix?

**Comment by ianobermiller** [comment]
I published 0.16.4 a few days ago, you should be good now.
